### PR TITLE
[Mijn-709] pre prod works

### PR DESCRIPTION
**Pull request by timvanoostrom**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -6,6 +6,7 @@ REACT_APP_API_BASE_URL=http://localhost:5000/api
 REACT_APP_LOGIN_URL=http://localhost:5000/api/login
 REACT_APP_LOGOUT_URL=http://localhost:5000/api/logout
 REACT_APP_ERFPACHT_URL=https://example.org/erfpachtdingen
+REACT_APP_ATLAS_API_BASE_URL=http://localhost:5000/api
```
Comment (BennievanderWel) [inline .env]: Why is this on localhost? This should always be api.data.amsterdam, I think.

File: src/hooks/api/api.hook.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { useState, useEffect, useReducer } from 'react';
+import { useState, useEffect, useReducer, useMemo } from 'react';
```
Comment (BennievanderWel) [inline src/hooks/api/api.hook.ts]: useMemo isn't used.

File: .env (hunk 1 above)
Comment (timvanoostrom) [reply .env]: Nope, I've mocked the atlas search in the mock api.

File: src/components/MyArea/MyArea.tsx [hunk 3]
```diff
@@ -40,28 +41,33 @@ export function MyAreaMap({ trackCategory, url }: MyAreaMapComponentProps) {
   useEffect(() => {
     trackItemPresentation(trackCategory, 'Embed_kaart');
   }, []);
-
-  return url ? (
+  console.log('MYAREA MAP');
```
Comment (drakonen) [inline src/components/MyArea/MyArea.tsx]: console.log

**Pull request merged**